sources/packages/backend/apps/test-db-seeding/src/db-seeding/aest/basic-seeding/create-aest-users.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { User } from "@sims/sims-db";
+import { createFakeUser } from "@sims/test-utils";
+import {
+  DataSeed,
+  DataSeedMethod,
+  SeedPriorityOrder,
+} from "../../../seed-executors";
+import { Repository } from "typeorm";
+import * as faker from "faker";
+
+@Injectable()
+@DataSeed({ order: SeedPriorityOrder.Priority1 })
+export class CreateAESTUsers {
+  constructor(
+    @InjectRepository(User)
+    private readonly userRepo: Repository<User>,
+  ) {}
+
+  /**
+   * Create AEST users used for tests.
+   */
+  @DataSeedMethod()
+  private async createAESTUsers(): Promise<void> {
```

Comment:
It seems weird to see a method decorated with `DataSeedMethod` as private. Even if it worked I would recommend making it public, removing the `private`.